images/build-e2e/common/filter.xsl

```diff
@@ -6,13 +6,13 @@ xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
 <!-- identity transform -->
 <xsl:template match="@*|node()">
     <xsl:copy>
-            <xsl:apply-templates select="@*|node()"/>
+        <xsl:apply-templates select="@*|node()"/>
     </xsl:copy>
 </xsl:template>
 
 <xsl:template match="failure">
     <xsl:copy>
-        <xsl:value-of select="@message"/>
+        <xsl:value-of select="substring(@message,0,1000000)"/>
```

Comment:
Is it possible to pick the latest chars from the failure message.. It would have more sense as typically you see the error at the very end

Reply:
if the error is at the end...

is there a way to link to the full report or a compressed form?

Reply:
So this made me think, the problem is about adding too much data on the junit files right? 

Because it seems there is another option to import full logs next to results....

If this is the case may we should see how to avoid the error log to be added on the junit file, for ginkgo AFAIK there are some customization you can do on the reporter to avoid the full trace... on the e2e side I think it is us adding the full error in code.

Can you check this @albfan

Reply:
While not by API, I try that solution from xml. tag `system-out` and `system-err` (under tag `test-case`) import logs.

they broke the web reportportal the same if they are big enough.

We ask our provider if there's a timeout on loading log view on reportportal.

Meanwhile we get a reply,  I will extract half the beginning and the end of message

Reply:
filter should now truncate in the middle of log showing first 50000 chars and last 500000 chars